tests/sync_diff_inspector/snapshot/run.sh

```diff
@@ -20,15 +20,28 @@ check_contains "sourceDB don't equal targetDB" $OUT_DIR/snapshot_diff.log
 # fix.sql will be empty after check below, so backup it
 cp $OUT_DIR/fix.sql $OUT_DIR/fix.sql.bak
 
-echo "use snapshot compare data, data should be equal"
+echo "use snapshot compare data, test sql mode by the way, will return error, diff should not passed"
+mysql -uroot -h 127.0.0.1 -P 4000 -e "SET GLOBAL sql_mode = 'ANSI_QUOTES';"
+sleep 10
+mysql -uroot -h 127.0.0.1 -P 4000 -e "show variables like '%sql_mode%'"
+mysql -uroot -h 127.0.0.1 -P 4000 -e "show create table diff_test.test"
```

Comment:
why do we need these lines?

Reply:
just for debug, set sql_mode will not take effect immediately in tidb, so need to wait some time and then check it